iwinfo_nl80211.c

```diff
@@ -153,6 +153,33 @@ static int nl80211_readstr(const char *path, char *buffer, int length)
 	return rv;
 }
 
+static bool nl80211_is_numeric(const char *str)
+{
+	return strlen(str) > 0 && !str[strspn(str, "0123456789")];
+}
+
+static bool nl80211_should_get_station(const char *req_name,
+                                       const char *cur_name)
+{
+	if (strncmp(cur_name, req_name, strlen(req_name)))
```

Comment:
I count 6 `strlen(req_name)` in this function. I'd say that's a reason to get it once and pass that var around ;)